ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml

```diff
@@ -33,6 +37,21 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: installer-rehearse-debug
+  interval: 960h
+  steps:
+    cluster_profile: aws-qe
+    post:
+    - as: online-debug
+      commands: sleep 7200
+      from: upi-installer
+      resources:
+        requests:
+          cpu: 200m
+          memory: 100Mi
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-provision
```

Comment:
Several enhance, 
```
- as: installer-rehearse-debug
  cron: '@yearly'
  steps:
    cluster_profile: aws-qe
    env:
      SLEEP_DURATION: 6h
    post:
    - ref: wait
    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
    pre:
    - chain: cucushift-installer-rehearse-aws-ipi-provision
```

Reply:
`wait` ref is using cli image, while `upi-installer` image have more tools installed, that would help the debugging, so I introduced a new step - `cucushift-installer-wait`

Others enhance is done.